test/test_api_usage.py

```diff
@@ -5061,5 +5067,40 @@ def test_delete_message(self):
         self.assertEqual(len(json["messages"]), 1)
 
 
+class TestOrganizationRhSku(ApiTestCase):
+    def test_bind_sku_to_org(self):
+        self.login(ADMIN_ACCESS_USER)
+        self.postResponse(
+            resource_name=OrganizationRhSku,
+            params=dict(orgname=ORGANIZATION),
+            data={"subscription_id": 12345},
+            expected_code=201,
+        )
+        json = self.getJsonResponse(
+            resource_name=OrganizationRhSku,
+            params=dict(orgname=ORGANIZATION),
+        )
+        self.assertEqual(len(json), 1)
```

Comment:
Can we add a test with an SKU that's already bound to an org, where it wouldn't return a 2XX?